examples/nlp/language_modeling/megatron_ckpt_to_nemo.py

```diff
@@ -104,7 +119,7 @@ def convert(rank, world_size, args):
 
 if __name__ == '__main__':
     args = get_args()
-    world_size = args.tensor_model_parallel_size
+    world_size = args.tensor_model_parallel_size * args.pipeline_model_parallel_size
 
     if args.local_rank == -1:
         device = torch.device("cuda" if torch.cuda.is_available() and not args.no_cuda else "cpu")
```

Comment:
Need multiple nodes support to convert large pipeline parallel checkpoints.
something like
https://github.com/NVIDIA/NeMo/blob/72eebf39930eed168bd2433b89d3dfb59c2889a3/examples/nlp/language_modeling/megatron_lm_ckpt_to_nemo.py#L389